eng/common/scripts/Helpers/Metadata-Helpers.ps1

```diff
@@ -80,3 +80,54 @@ function GetPrimaryCodeOwner ([string]$TargetDirectory)
     Write-Warning "No code owner found in $TargetDirectory."
     return $null
 }
+
+function GetDocsMsService($clientPackageInfo, $mgmtPackageInfo, $serviceName) 
```

Comment:
We split `mgmt` and `client` packages at the surface level, then pass in both arrays of package info. However, I'm not certain I agree that they really NEED to be split up. 

I can totally get avoiding multiple `query set for client packages`, `query set for mgmt packages` situations...but we don't actually _look_ at them until we pass them to generateMarkdown table. At least, not outside of this function where we take the first data package's service (falling back to first mgmt packages service). 

All that being said, I'm not certain it's worth turning the entire PR on it's head just to make this a single set that is filtered when you need it 👍